docs/deploying/spacetimedb-standalone.md

```diff
@@ -82,25 +82,67 @@ server {
     listen 80;
     server_name example.com;
 
+    #########################################
+    # Warning: By default SpacetimeDB is completely open so that anyone can publish to it. If you want to block
+    # users from creating new databases you should comment out this section. We generally would recommend
+    # commenting this section out and then only publishing your changes locally.
+    #########################################
     location / {
         proxy_pass http://localhost:3000;
         proxy_http_version 1.1;
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection "Upgrade";
         proxy_set_header Host $host;
+        break;
     }
 
-    # This restricts who can publish new databases to your SpacetimeDB instance. We recommend
-    # restricting this ability to local connections. 
-    location /v1/publish {
-        allow 127.0.0.1;
-        deny all;
+    # Anyone can subscribe to any database.
+    # Note: This is the only section *required* for the websocket to function properly. Clients will
+    # be able to create identities, call reducers, and subscribe to tables through this websocket.
+    location ~ ^/v1/[^/]+/subscribe$ {
         proxy_pass http://localhost:3000;
         proxy_http_version 1.1;
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection "Upgrade";
         proxy_set_header Host $host;
     }
+
+    # Uncomment this section to allow all HTTP reducer calls
+    # location ~ ^/v1/[^/]+/call/[^/]+$ {
+    #     proxy_pass http://localhost:3000;
+    #     proxy_http_version 1.1;
+    #     proxy_set_header Upgrade $http_upgrade;
+    #     proxy_set_header Connection "Upgrade";
+    #     proxy_set_header Host $host;
+    # }
+
+    # Uncomment this section to allow all HTTP sql requests
+    # location ~ ^/v1/[^/]+/sql$ {
+    #     proxy_pass http://localhost:3000;
+    #     proxy_http_version 1.1;
+    #     proxy_set_header Upgrade $http_upgrade;
+    #     proxy_set_header Connection "Upgrade";
+    #     proxy_set_header Host $host;
+    # }
+
+    # Uncomment this section to optionally allow /v1/identity. This would allow remote users to create
+    # an identity for the purpose of connecting to your SpacetimeDB instance.
+    # NOTE: this is *not* required for the subscribe route to function.
+    # location /v1/identity {
+    #     proxy_pass http://localhost:3000;
+    #     proxy_http_version 1.1;
+    #     proxy_set_header Upgrade $http_upgrade;
+    #     proxy_set_header Connection "Upgrade";
+    #     proxy_set_header Host $host;
+    # }
+
+    # Block all other routes explicitly. Only localhost can use these routes.
+    # We would recommend uncommenting this section when you don't want remote users to
+    # publish to your database!
+    # location / {
+    #     allow 127.0.0.1;
+    #     deny all;
+    # }
```

Comment:
we have lines below that say:
> This configuration contains a restriction to the `/v1/publish` route. This restriction makes it so that you can only publish to the database if you're publishing from a local connection on the host. 

Do we want to update that since that block isn't enabled by default anymore? (Maybe just removing it entirely, since I think the inline comments do a pretty good job).

Reply:
*Interesting*, I thought the default before was open. I will gladly changed the default to not be open and I'll update that paragraph 👍

Reply:
(I think we still want to upgrade the paragraph at the very least to not say `/v1/publish` since we don't explicitly touch a `publish` route anymore)